distribution/lib/Standard/Table/0.0.0-dev/src/Data/Column.enso

```diff
@@ -1327,29 +1324,19 @@ run_vectorized_unary_op column name fallback_fn =
     Column_Data (Java_Column.new "Result" ix rs)
 
 ## PRIVATE
-
-   Keep this in sync with `org.enso.table.data.Storage.Type.LONG`
-storage_type_long : Integer
-storage_type_long = 1
-
-## PRIVATE
-
-   Keep this in sync with `org.enso.table.data.Storage.Type.DOUBLE`
-storage_type_double : Integer
-storage_type_double = 2
+   Enumerates storage types in a way that is consistent with
+   `org.enso.table.data.Storage.Type`, i.e.
+   `storage_type.at org.enso.table.data.Storage.Type.LONG` will yield the
+   corresponding `Storage.Integer`.
+storage_types : Vector Storage.Type
+storage_types = [Storage.Any, Storage.Integer, Storage.Decimal, Storage.Text, Storage.Boolean, Storage.Date, Storage.Time_Of_Day, Storage.Date_Time]
 
 ## PRIVATE
 
    Keep this in sync with `org.enso.table.data.Storage.Type.STRING`
 storage_type_string : Integer
```

Comment:
Hmmm, I think you are right, although please note that this will change the semantics.

Let's consider a table:

<table>
<tr>
	<th> mixed
	<th> strings
<tr>
	<td> 4
	<td> b
<tr>
	<td> a
	<td> c
</table>

Then:
- `get_item_string (table.at "mixed") 0 == "4"`
- `get_item_string (table.at "mixed") 1 == "'a'"`
- `get_item_string (table.at "mixed") 0 == "b"`

Note how, because we call `string.to_text` the `a` gets wrapped in additional quotes.

Although, I think `string.to_text` was supposed to be changed at some point to be identity on strings? Then it will not matter indeed. Also maybe it is actually better for this to be consistent for all strings regardless of if they are stored in a special Text column or in a mixed one.

So I will change it, with a note that later on we can even remove this branch and just keep `column.getItem ix . to_text`.